backend/migrate_imports_dates.py

```
@@ -0,0 +1,161 @@
+            
+            # Check if there are more pages
+            if 'LastEvaluatedKey' not in response:
+                break
+                
+            response = table.scan(ExclusiveStartKey=response['LastEvaluatedKey'])
+        
+        logger.info(f"Found {len(accounts)} accounts total")
+        return accounts
+        
+    except ClientError as e:
+        logger.error(f"Error scanning accounts table: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)